src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Shop;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Webmozart\Assert\Assert;
+
+final class ProductContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var ResponseCheckerInterface */
+    private $responseChecker;
+
+    public function __construct(ApiClientInterface $client, ResponseCheckerInterface $responseChecker)
+    {
+        $this->client = $client;
+        $this->responseChecker = $responseChecker;
+    }
+
+    /**
+     * @When /^I check (this product)'s details$/
+     */
+    public function iOpenProductPage(ProductInterface $product): void
+    {
+        $this->client->show($product->getSlug());
+    }
+
+    /**
+     * @Then I should see the product name :name
+     */
+    public function iShouldSeeProductName(string $name): void
+    {
+        Assert::true($this->responseChecker->hasItemWithTranslation(
+            $this->client->getLastResponse(),
+            'en_US',
+            'name',
+            $name))
+        ;
```

Comment:
```suggestion
        Assert::true(
            $this->responseChecker->hasItemWithTranslation($this->client->getLastResponse(), 'en_US', 'name', $name)
        );
```
?

Reply:
I chose solution between, because your proposition was too long
<img width="1050" alt="Screenshot 2020-07-22 at 12 37 53" src="https://user-images.githubusercontent.com/39232096/88167422-16622180-cc19-11ea-83c5-8cd018de09d2.png">